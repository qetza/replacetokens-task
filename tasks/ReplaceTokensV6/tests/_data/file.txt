var.var: #{var.VAR}#
var.secret: #{VAR_secret}#
var1: #{VAR1}#
var2: #{VAR2}#
var_json: #{var_json}#
var_yaml1: #{var_yaml1}#
var_yaml2: #{var_yaml2}#
var.yml1: #{var.yml1}#
var.yml2: #{var.yml2}#
unknown: #{unknown}#