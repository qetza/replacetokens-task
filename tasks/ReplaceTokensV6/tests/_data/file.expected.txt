var.var: var
var.secret: secret
var1: env
var2: inline
var_json: file
var_yaml1: file
var_yaml2: file
var.yml1: file
var.yml2: inline
unknown: DEFAULT